src/app.ts

```diff
@@ -15,12 +17,15 @@ export async function createApp({
   messageBroker: MessageBroker;
 }): Promise<Application> {
   const app = express();
+  const keycloak = new Keycloak({}, config.get('keycloak'));
```

Comment:
Move Keycloak to own file, so that it can be imported where need be, instead of being passed around.

Reply:
fixed